### DISCUSSION TITLE
is rebalance available in opensource emqx version 5.0.24?

### DISCUSSION THREAD

**Discussion by haripriyapendem** [emqx/emqx#13251] [Q&A] [answered]
Hello, is load rebalancing available in open source emqx:5.0.24 version. I am seeing help options when i try to rebalance in my cluster. Could you please explain why it is displaying help response when i execute `./bin/emqx_ctl rebalance start --evacuation \
	--wait-health-check 60 \
	--wait-takeover 200 \
	--conn-evict-rate 30 \
	--sess-evict-rate 30 \
	--migrate-to "emqx2@127.0.0.1 emqx3@127.0.0.1"`

**Comment by zmstone** [comment, marked as answer]
no. it's enterprise-only feature
